src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -1616,6 +1740,20 @@ public ObjectSpec GetObjectSpec(Rand rand)
         bool shouldBePinned = shouldSurvive && Util.isNth(pinInterval, count / survInterval);
         bool shouldBeFinalizable = shouldSurvive && Util.isNth(finalizableInterval, count / survInterval);
 
+        if (isPoh)
+        {
+            sohAllocatedBytes += overhead;
+            pohAllocatedBytes += (size - overhead);
+        }
+        else if (size >= 85000)
```

Comment:
you might want to not hard code this -  you could be running with the LOH threshold changed with the config